Apps/Sandcastle/.eslintrc.json

```diff
@@ -11,7 +11,6 @@
         "Cesium": true
     },
     "parserOptions": {
-        "ecmaVersion": 2015,
         "sourceType": "script"
     },
     "rules": {
```

Comment:
Of these, I was able to remove `quotes`, but two others seem necessary to the way Sandcastles are currently written:

- `no-alert`: Used frequently in things like `FeatureDetection` within Sandcastles
- `no-unused-vars`: Simply declaring the `viewer` in an "empty" Sandcastle will cause an error here. (99 problems exist)

For the last one:
- `no-implicit-globals`: Since we already read the `startup` function as a property of the window in `load-cesium-es6.js`:
https://github.com/CesiumGS/cesium/blob/82465d52c6b41018cf16677cb18c56c05194dea3/Apps/Sandcastle/load-cesium-es6.js#L10

I figured setting the function to that directly would be the right fix. Sandcastle still works as intended, but please let me know if this is not the right fix here, introduced in [this commit](https://github.com/CesiumGS/cesium/pull/10396/commits/a4a3b783d835c20c46e06db88b3bb3d30bf2bfcf).